profanity.rb

```diff
@@ -1640,6 +1647,19 @@ def get_color_pair_id(fg_code, bg_code)
             elsif RbConfig::CONFIG['host_os'] =~ /linux|bsd/
               system "xdg-open #{url} >/dev/null 2>&1 &"
             end
+					elsif xml =~ /^<a/
+						if blue_links						
+							h = { :start => start_pos }
+							h[:fg] = '6060ff'
+							h[:bg] = nil
```

Comment:
Should be set to user-definable via xml template:
```ruby
							h[:fg] = PRESET['links'][0]
							h[:bg] = PRESET['links'][1]
```
Then just need to edit the default xml to have the preset defined:
https://github.com/elanthia-online/ProfanityFE/blob/master/templates/default.xml
```xml
  <preset id='links' fg='6060ff'/>
```